x11-packages/weston/bump-neatvnc.patch

```diff
@@ -1,10 +1,10 @@
 --- a/libweston/backend-vnc/meson.build
 +++ b/libweston/backend-vnc/meson.build
-@@ -3,7 +3,7 @@ if not get_option('backend-vnc')
+@@ -3,7 +3,7 @@
  endif
  
  config_h.set('BUILD_VNC_COMPOSITOR', '1')
--dep_neatvnc = dependency('neatvnc', version: ['>= 0.7.0', '< 0.8.0'], required: false, fallback: ['neatvnc', 'neatvnc_dep'])
+-dep_neatvnc = dependency('neatvnc', version: ['>= 0.7.0', '< 0.9.0'], required: false, fallback: ['neatvnc', 'neatvnc_dep'])
 +dep_neatvnc = dependency('neatvnc', version: ['>= 0.7.0'], required: false, fallback: ['neatvnc', 'neatvnc_dep'])
```

Comment:
This patch may not be required. neatvnc is version 0.8.1 which is in the required version range.